subsys/bluetooth/mesh/Kconfig

```diff
@@ -103,6 +103,64 @@ menuconfig BT_MESH_ADV_EXT
 
 if BT_MESH_ADV_EXT
 
+choice BT_MESH_WORKQ_CONTEXT
+	prompt "Advertising thread selection"
+	default BT_MESH_WORKQ_MESH
+	help
+	  Defines a context for mesh messages transmission.
+
+config BT_MESH_WORKQ_MESH
+	bool "Send messages from in the mesh-specific workqueue"
```

Comment:
Consider reducing this title to:
```suggestion
	bool "Mesh-specific workqueue"
```

This will thing look neater in menuconfig, which is where the title has a role.
Without change:
![image](https://github.com/user-attachments/assets/a7f010ac-0782-45b0-9360-c14b463104b6)

With change:
![image](https://github.com/user-attachments/assets/42b56034-f7e4-4d16-a7bf-633534d18051)